### DISCUSSION TITLE
Wrap text [Aug 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6245] [Discord Threads]
> 🙋‍♂️ **.tucado**  
## Hi Everyone. Is it possible to wrap text within grid view? What I meant by this is to have multiple elements broken into the new line when they don't fit. Currently it seems to be impossible  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1130844900016013404/image.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### you can use column height  
#  
> 🙋‍♂️ **.tucado**  
#### it doesn't help, I'm afraid  
![image0](https://cdn.discordapp.com/attachments/1130844900246683740/1131580121682280598/image.png)  
#  
> 💁 **o1lab**  
#### <@190550716497133568>  
#  
> 🙋‍♂️ **mertm1t**  
#### can you please share your project info? (https://docs.nocodb.com/faqs/#how-to-check-my-project-info-)   
#### also which browser are you using?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1130844900246683740)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
